airbyte-integrations/connectors/source-mailjet-mail/source_mailjet_mail/mailjet_mail.yaml

```diff
@@ -0,0 +1,91 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["Data"]
+  requester:
+    url_base: "https://api.mailjet.com/v3/REST"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['username'] }}"
+      password: "{{ config['password'] }}"
+  offset_paginator:
+    type: DefaultPaginator
+    $options:
+      url_base: "*ref(definitions.requester.url_base)"
+    pagination_strategy:
+      type: "OffsetIncrement"
+      page_size: 50
+    page_token_option:
+      field_name: "offset"
+      inject_into: "request_parameter"
+    page_size_option:
+      inject_into: "request_parameter"
+      field_name: "limit"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  contactslist_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "contactslist"
+      primary_key: "ID"
+      path: "/contactslist?limit=100"
+  contacts_stream:
+    $ref: "*ref(definitions.base_stream)"
+    retriever:
+      record_selector:
+        $ref: "*ref(definitions.selector)"
+      paginator:
+        $ref: "*ref(definitions.offset_paginator)"
+      requester:
+        $ref: "*ref(definitions.requester)"
+    $options:
+      name: "contacts"
+      primary_key: "ID"
+      path: "/contact"
+  stats_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "stats"
+      primary_key: "APIKeyID"
+      path: "/statcounters?CounterSource=APIKey&CounterResolution=Lifetime&CounterTiming=Message"
```

Comment:
This is to request a specific stats?

Reply:
I changed the stream name, and yes it's a specific stats (Mailjet API design is weird :) )